annotation-processor/src/main/java/org/hibernate/validator/ap/ConstraintAnnotationVisitor.java

```diff
@@ -112,6 +113,23 @@ public Void visitVariableAsField(VariableElement annotatedField, List<Annotation
 	/**
 	 * <p>
 	 * Checks whether the given annotations are correctly specified at the given
+	 * method parameter. The following checks are performed:
+	 * </p>
+	 * <ul>
+	 * <li>
+	 * Overridden method parameters cannot have wider range of constraints.
+	 * </li>
+	 * </ul>
+	 */
+	@Override
+	public Void visitVariableAsParameter(VariableElement annotatedField, List<AnnotationMirror> mirrors) {
```

Comment:
These changes have already been merged, haven't they? If so, if you rebase your PR, they should be included. You might have to solve some conflicts though. When you rebase, git status gives you useful hints on what you should fix and what you should do.